libs/state/src/lib/rx-state.service.ts

```diff
@@ -60,22 +66,24 @@ export type ProjectValueReducer<T, K extends keyof T, V> = (
  * @docsPage RxState
  */
 @Injectable()
-export class RxState<T extends object> implements OnDestroy, Subscribable<T> {
+export class RxState<TType extends object>
```

Comment:
I would name the generic `TState` instead.

Reply:
I'm waiting with this fix until we will be sure with naming convention for generics, from the other hand happy christmas and new year to everyone :)